styles.css

```diff
@@ -82,12 +82,12 @@ header {
 }
 
 #social_media {
+  color: #d3e1f9;
```

Comment:
- [ ] [OPTIONAL] It would be better if you change the background color of the headline as in the Figma design in order to match it 
- Yours 
<img width="364" alt="Screen Shot 2022-10-01 at 21 39 37" src="https://user-images.githubusercontent.com/61790587/193427551-335a4af0-8375-4674-b54d-e38c65bd7cae.png">
- Figma

![](https://user-images.githubusercontent.com/69077061/193378589-adea8ce8-4eff-4a8f-873c-15aefd5a3b83.PNG)

Reply:
@youmari Thanks for the review, your suggestion is noted and will be effected.
cheers.